docs/data/data-grid/pivoting/pivoting.md

```diff
@@ -4,17 +4,15 @@ title: Data Grid - Pivoting
 
 # Data Grid - Pivoting [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')🚧
 
-<p class="description">Turn a column values into columns.</p>
+<p class="description">Rearrange rows and columns to view data from multiple perspectives.</p>
 
-:::warning
-This feature isn't implemented yet. It's coming.
-
-👍 Upvote [issue #214](https://github.com/mui/mui-x/issues/214) if you want to see it land faster.
-
-Don't hesitate to leave a comment on the same issue to influence what gets built. Especially if you already have a use case for this component, or if you are facing a pain point with your current solution.
+:::success
+Pivoting is available in MUI X v8+.
```

Comment:
WDYT about linking to the feature on v8 as well? 🤔

Reply:
No strong preference here.
If you think that it hinders UX, I'm fine and agree that the main actionable item is the upgrade guide. 👌